pkg/action/get_metadata.go

```diff
@@ -28,14 +28,15 @@ type GetMetadata struct {
 }
 
 type Metadata struct {
-	Name       string `json:"name" yaml:"name"`
-	Chart      string `json:"chart" yaml:"chart"`
-	Version    string `json:"version" yaml:"version"`
-	AppVersion string `json:"appVersion" yaml:"appVersion"`
-	Namespace  string `json:"namespace" yaml:"namespace"`
-	Revision   int    `json:"revision" yaml:"revision"`
-	Status     string `json:"status" yaml:"status"`
-	DeployedAt string `json:"deployedAt" yaml:"deployedAt"`
+	Name        string            `json:"name" yaml:"name"`
+	Chart       string            `json:"chart" yaml:"chart"`
+	Version     string            `json:"version" yaml:"version"`
+	AppVersion  string            `json:"appVersion" yaml:"appVersion"`
+	Namespace   string            `json:"namespace" yaml:"namespace"`
+	Revision    int               `json:"revision" yaml:"revision"`
+	Status      string            `json:"status" yaml:"status"`
+	DeployedAt  string            `json:"deployedAt" yaml:"deployedAt"`
+	Annotations map[string]string `json:"annotations,omitempty" yaml:"annotations,omitempty"`
```

Comment:
This outputs as `ANNOTATIONS: map[key0:val0 key1:val1 ...etc]`
Perhaps taking the same approach as `get_values.go` which outputs the map as yaml might be appropriate?